### DISCUSSION TITLE
How can I redirect start() function's output?

### DISCUSSION THREAD

**Discussion by justjew** [onepub-dev/dcli#226] [Q&A] [answered]
I am using DCLI to make my white-lable's CLI.
There is `publish` command which is cleaning, building and publishing a pack of Flutter applications.
So this command runs a lot of console programs and all those outputs are printing to terminal.
I would like to redirect those outputs to a .log file, but keep my own info printings as default output.
Is it possible?

p.s.
hope I explained clearly

**Comment by bsutton** [comment, marked as answer]
Have a read of this:
https://app.gitbook.com/o/ecaaU3OrP2aDt9qZRymk/s/-MGR3_6kTBSdRgC-z--_/dcli-api/redirecting-output

**Reply by justjew** [reply]
That is exactly what I needed. Thank you!
